modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/utils/settingsForm.es.js

```diff
@@ -89,6 +89,14 @@ export const getFilteredSettingsContext = ({
 						] = getPredefinedValues(field);
 					}
 
+					if (fieldName == 'repeatable') {
```

Comment:
@carolmariaabb, is there any reason for not use the operator `===` instead?

If not, can you please use it?

Reply:
No reason, I just didn't pay attention 😅 

I resent it here. Thanks!